src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -275,6 +275,10 @@ private void RaiseNoLivingNodesAlert(string alertMsg)
             }
 
             RemoveOtherNodesIfNeeded(dbName, topology, ref deletions);
+
+            if (topology.Stamp.Index == -1)
```

Comment:
Without this we are staying with Etag = -1 because we are not updating topology at all, so client does not update the topology also.